docs/_instructions/define-contract.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+class: Project
+title: -define-contract 
+summary: Define a contract when one cannot be added to the buildpath. 
+---
+
+Used in conjunction with the [`-contract`](contract.html) instruction,`-define-contract` can be applied in order to define a contract which is not available on the build path (i.e. compile class path).
+
+The contract specification is exactly the same syntax used in the `Provide-Capability` header.
+
+```properties
+-define-contract:\
+  osgi.contract;\
+    osgi.contract=Servlet;\
```

Comment:
Isn't the correct name `JavaServlet`? And the uses for JavaServlet 3.0 is "javax.servlet, javax.servlet.annotation, javax.servlet.descriptor, javax.servlet.http". https://www.osgi.org/portable-java-contract-definitions/